src/perf/telemetry.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * @license
+ * Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import * as now from 'performance-now';
+
+export interface Measurement {
+  elapsedTime: number;
+  kind: string;
```

Comment:
"parse" is a kind "./bower_components/paper-button/paper-button.html" is an identifier